features/importing/work_import.feature

```diff
@@ -220,3 +220,23 @@ Feature: Import Works
     When I import "http://www.intimations.org/fanfic/idol/Huddling.html"
     Then I should see "Preview"
       And I should see "English"
+
+  Scenario: Searching for a work url should be cached
+    Then there should be no work_url cached for "http://www.scarvesandcoffee.net/viewstory.php?sid=9570"
```

Comment:
Wouldn't it make more sense to actually _empty_ the cache at the start of the test, rather than just _check_ that it's empty? So like:
```
Given /^there are no cached work_urls for "([^"]*)"$/ do |url|
  Rails.cache.delete(Work.find_by_url_cache_key(url))
end
```
And then leave all the `Then there should be (no|a) work_url cached for` bits alone.